dev/io.openliberty.org.jboss.resteasy.common/resources/io/openliberty/org/jboss/resteasy/common/nls/RESTfulWSServer.nlsprops

```diff
@@ -68,3 +68,11 @@ INVALID_INT_PROPERTY_CWWKW1303W=CWWKW1303W: The {0} client property must be an i
 INVALID_INT_PROPERTY_CWWKW1303W.explanation=Client properties, such as proxy port settings, must be configured as a positive integer or a string that can be converted to an integer. This message indicates that a client property was configured with an invalid value. This property will be ignored.
 INVALID_INT_PROPERTY_CWWKW1303W.useraction=Review the application code and server configuration to determine where this property was set incorrectly and either remove the property or change it to a valid integer value.
 
+INVALID_BOOLEAN_PROPERTY_CWWKW1304W=CWWKW1304W: The {0} client property must be an boolean, but was configured incorrectly with {1}. This setting will be ignored.
+INVALID_BOOLEAN_PROPERTY_CWWKW1304W.explanation=Client properties, such as whether to automatically follow redirects, must be configured as a boolean or a string that can be converted to an boolean. This message indicates that a client property was configured with an invalid value. This property will be ignored.
```

Comment:
- Change "an boolean" to "a boolean" (change "an" to "a").
- Change "was configured with" to "is set to"